src/messages/javascript.js

```diff
@@ -124,26 +124,24 @@ export const UNSUPPORTED_API = {
   description: i18n._('This API has not been implemented by Firefox.'),
 };
 
-function deprecatedAPI(api) {
-  return {
-    code: apiToMessage(api),
-    message: i18n._(`"${api}" is deprecated or unimplemented`),
-    description: i18n._(oneLine`This API has been deprecated by Chrome
-      and has not been implemented by Firefox.`),
-  };
-}
+export const DEPRECATED_API = {
+  code: 'DEPRECATED_API',
+  message: null,
+  messageFormat: i18n._('{{api}} is deprecated'),
+  description: i18n._('This API has been deprecated by Firefox.'),
+};
 
-export const APP_GETDETAILS = deprecatedAPI('app.getDetails');
-export const EXT_ONREQUEST = deprecatedAPI('extension.onRequest');
-export const EXT_ONREQUESTEXTERNAL = deprecatedAPI(
-  'extension.onRequestExternal'
-);
-export const EXT_SENDREQUEST = deprecatedAPI('extension.sendRequest');
-export const TABS_GETALLINWINDOW = deprecatedAPI('tabs.getAllInWindow');
-export const TABS_GETSELECTED = deprecatedAPI('tabs.getSelected');
-export const TABS_ONACTIVECHANGED = deprecatedAPI('tabs.onActiveChanged');
-export const TABS_ONSELECTIONCHANGED = deprecatedAPI('tabs.onSelectionChanged');
-export const TABS_SENDREQUEST = deprecatedAPI('tabs.sendRequest');
+export const DEPRECATED_CHROME_API = {
+  // We are re-using the same code here for consistency and for technical
+  // reasons. We aren't really able to issue different codes from the same
+  // rule, so until we have to, we're going to re-use the `DEPRECATED_API`
+  // code.
+  code: 'DEPRECATED_API',
+  message: null,
+  messageFormat: i18n._('"{{api}}" is deprecated or unimplemented'),
+  description: i18n._(oneLine`This API has been deprecated by Chrome
```

Comment:
Nit, this is currently unused (the linting warnings/error produced will have the messageFormat specified here, but the description isn't currently overwritten), it may be worth to leave an inline comment here (and possibly mention a github issue to look into it at some point).

Otherwise, e.g. if we don't think it is worth to be fixed in a followup, could we just remove the unused description property?

Reply:
Fixed (accidentally) as part of my merge in d7ff5e57, going to rebase before merging anyway.